Library/Formula/silk.rb

```diff
@@ -17,8 +15,12 @@ def install
                           "--enable-ipv6",
                           "--enable-data-rootdir=#{var}/silk"
     system "make"
-    system "make install"
+    system "make", "install"
 
     (var+"silk").mkpath
   end
+
+  test do
+    system "#{bin}/silk_config", "--version"
```

Comment:
Added an enhanced test. I don't like it because it stores `test.pcap` fixture converted to ipfix using yaf and stored in the formula as base64. Probably it can be implemented better, too.